### PR TITLE
ci-bootstrap: Don't fail on things already installed

### PR DESCRIPTION
**Pull request by vinzenz**

### CODE REVIEW

**Comment by shaded-enmity** [comment]
I don't like the `|| :` stuff because while it can solve the problem of already installed packages it also *masks* every other form of error. At least in the Ansible roles we should first save the installed/not-installed status in a register and add `when:` conditional. 
As for `yum` I think this is OK, since return codes of `yum` are just grotesquely inconsistent.

File: centos-ci/ansible/roles/vagrant/tasks/main.yml [hunk 1]
```diff
@@ -29,26 +29,26 @@
     - nmap
 
 - name: Run VMs under root
-  lineinfile: 
+  lineinfile:
```
Comment (vinzenz) [inline centos-ci/ansible/roles/vagrant/tasks/main.yml]: The only thing I did here was to remove the trailing space

**Comment by ncoghlan** [comment]
Like @shaded-enmity, I don't like the fact that the approach here hides all `yum` errors, not just "package already installed" ones.

Perhaps we could just pass the `-t` switch to yum, so it doesn't treat "already installed" as an error?

**Comment by ncoghlan** [comment]
Also, it would be good to separate the functional change here from the formatting change that strips the trailing spaces from the various files.

**Pull request closed without merging**